internal/deb/extract.go

```diff
@@ -246,11 +246,17 @@ func extractData(dataReader io.Reader, options *ExtractOptions) error {
 				}
 			}
 			// Create the entry itself.
+			link := tarHeader.Linkname
+			if tarHeader.Typeflag == tar.TypeLink {
+				// The hard link requires the real path of the target file.
```

Comment:
```suggestion
				// A hard link requires the real path of the target file.
```

Reply:
Fixed in [03ce4d3](https://github.com/letFunny/chisel/pull/17/commits/03ce4d322ccaff4f6ba8c00a32aee039de12c9d3).